src/components/campaigns/CampaignSelect.tsx

```diff
@@ -45,19 +48,21 @@ export default function CampaignSelect({
         select
         type="text"
         fullWidth
-        defaultValue=""
+        defaultValue={selectedCampaign}
         label={t(label)}
         {...textFieldProps}
         {...field}
         onChange={handleChange}>
-        <MenuItem value="" disabled>
-          {t(label)}
-        </MenuItem>
-        {campaigns?.map((value, index) => (
-          <MenuItem key={index} value={value.id}>
-            {value.title}
-          </MenuItem>
-        ))}
+        {campaigns
+          ?.filter((value) => {
+            return value.vaults && value.vaults.length > 0
+          })
+          .map((value, index) => (
+            //select the element that matches the selectedCampaign
+            <MenuItem key={index} value={value.id} selected={value.id === selectedCampaign}>
+              {value.title} ({value.id}) s({value.id === selectedCampaign})
```

Comment:
Just interested why is this `s({value.id === selectedCampaign})`. Why do we have the s()

Reply:
Leftover from debug. 
Removed it.